test/py/onnx_backend_test.py

```diff
@@ -143,7 +143,6 @@ def disabled_tests_onnx_1_7_0(backend_test):
     backend_test.exclude(r'test_resize_upsample_sizes_cubic_cpu')
     backend_test.exclude(r'test_reversesequence_batch_cpu')
     backend_test.exclude(r'test_reversesequence_time_cpu')
-    backend_test.exclude(r'test_scan9_sum_cpu')
     backend_test.exclude(r'test_scan_sum_cpu')
```

Comment:
did you test without the test_scan_sum_cpu as well? Shouldn't we be able to remove both from the exlude list now?

Reply:
That test uses the opset 8 version of Scan which is unsupported by this implementation. The reason being that the version 8 modus operandi is quite different compared to all the later version of the operator.  https://onnx.ai/onnx/operators/onnx__Scan.html#scan-8